Rakefile

```diff
@@ -165,7 +165,17 @@ def _flatten_project
 
     puts "Working on #{Dir[dir + '/*.tf']}" if debug
     system("terraform get #{dir}")
-    FileUtils.cp( Dir["#{dir}/*.tf"], tmp_dir)
+    FileUtils.cp(Dir["#{dir}/*.tf"], tmp_dir)
+
+    # this has a small chance of overwriting a file if both resources/ and resources/$env
+    # have the same file
+    ['files', 'templates'].each do |subdir|
+      if File.directory? "#{dir}/#{subdir}/"
```

Comment:
This works for a directory structure as per @surminus's #46 comment if we also pass the project directory into the loop at line line 163